tests/select-ranges.test.ts

```diff
@@ -34,6 +34,10 @@ describe('live demo', () => {
   beforeEach(async () => {
     const chromeOptions = new chrome.Options();
     chromeOptions.addArguments('--disable-web-security');
+    if(process.env.CHROME_BIN_PATH) {
```

Comment:
inconsistent formatting, don't we have an automatic code formatter for this repo?